### PR TITLE
[PLAYER-4463]: Fix cc alignment

### PR DESCRIPTION
**Pull request by rnegov**

### CODE REVIEW

**Comment by DmitriiMaslov** [comment]
Please provide a PR description

**Comment by rnegov** [comment]
Fix of PLAYER-4463 task connected to subtitle text is shifted to the left

**Review by MarkEPAM — APPROVED** [review]
LGTM

**Review by MarkEPAM — CHANGES_REQUESTED** [review]
Check watermark visibility

File: sdk/react/panels/style/panelStyles.json [hunk 1]
```diff
@@ -37,29 +35,29 @@
...
     "fontWeight": "bold",
     "fontSize": 16,
     "fontFamily": "Helvetica",
     "color": "white",
-    "backgroundColor": "rgba(0, 0, 0, 0.7)",
+    "backgroundColor": "rgba(0, 0, 0, 0.7)"
   },
   "closedCaptionsFlexibleSpace" : {
-    "flex":1,
+    "flex":1
```
Comment (MarkEPAM) [inline sdk/react/panels/style/panelStyles.json]: space between `"flex":` and `1`

File: sdk/react/panels/style/panelStyles.json [hunk 2]
```diff
@@ -37,29 +35,29 @@
...
-    "margin":2,
+    "margin": 2,
     "fontWeight": "bold",
     "fontSize": 16,
     "fontFamily": "Helvetica",
     "color": "white",
-    "backgroundColor": "rgba(0, 0, 0, 0.7)",
+    "backgroundColor": "rgba(0, 0, 0, 0.7)"
   },
   "closedCaptionsFlexibleSpace" : {
```
Comment (MarkEPAM) [inline sdk/react/panels/style/panelStyles.json]: remove space between `"closedCaptionsFlexibleSpace"` and `:`

Comment (rnegov) [reply sdk/react/panels/style/panelStyles.json]: fixed

File: sdk/react/panels/style/panelStyles.json (hunk 1 above)
Comment (rnegov) [reply sdk/react/panels/style/panelStyles.json]: fixed

**Review by ipadilla4 — APPROVED** [review]

**Review by MarkEPAM — APPROVED** [review]

**Pull request closed without merging**